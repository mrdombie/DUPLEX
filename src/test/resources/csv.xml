<?xml version="1.0" encoding="UTF-8"?>
<dataset>
	
	<csv_status status="Pending"/>
	<csv_status status="Failed"/>
	<csv_status status="Holding"/>
	<csv_status status="Processed"/>

	<csv id="1"/>
	<csv_entry entry_id="1" csv_id="1" name="tim" age="22" height="150" status="HOLDING" />				
		
</dataset>