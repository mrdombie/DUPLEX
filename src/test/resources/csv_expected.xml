<?xml version="1.0" encoding="UTF-8"?>
<dataset>
	
	<!-- 1	Pending
2	Failed
3	Holding
4	Processed -->
	
	<csv_status id="1" status="Pending"/>
	<csv_status id="2" status="Failed"/>
	<csv_status id="3" status="Holding"/>
	<csv_status id="4" status="Processed"/>
			
	<csv id = "1"/>
	<csv id = "2"/>
		
	<csv_entry entry_id="1" csv_id="1" name="tim" age="22" height="150" status="HOLDING" />		
	<csv_entry entry_id="2" csv_id="2" name="dom" age="33" height="186" status="PENDING" />		
	
</dataset>